css/at-rules/media.json

```diff
@@ -612,15 +612,7 @@
             "spec_url": "https://drafts.csswg.org/mediaqueries-5/#forced-colors",
             "support": {
               "chrome": {
-                "version_added": "79",
-                "flags": [
-                  {
-                    "type": "preference",
-                    "name": "#forced-colors",
-                    "value_to_set": "Enabled"
-                  }
-                ],
-                "notes": "See <a href='https://crbug.com/970285'>bug 970285</a>."
+                "version_added": "89"
```

Comment:
I am very happy to see flag data removed, but the [guideline](https://github.com/mdn/browser-compat-data/blob/master/docs/data-guidelines.md#removal-of-irrelevant-flag-data) is to do so only after the feature has been enabled by default for two years. So, what you'll need to do here instead is to use an array of two support statements, also adding `"version_added": "89"` for the flags entry.

Reply:
Apologies for missing the guidance (should have read more closely). This is fixed now.

Reply:
I doubt anyone reads the guidelines before a specific one is relevant in a PR, I haven't read all of them at least :)